test/test_browsing.py

```diff
@@ -0,0 +1,12 @@
+import os
+from dotenv import load_dotenv  # pip install python-dotenv
+from gpt4_openai import GPT4OpenAI
+load_dotenv()
+
+# Token is the __Secure-next-auth.session-token from chat.openai.com
+llm = GPT4OpenAI(token=os.environ["OPENAI_SESSION_TOKEN"],
+                 headless=False, model='gpt-4-browsing')
```

Comment:
ah yes, browsing came back :) does this work as expected?

Reply:
Yup